ansible/roles/images/tasks/load.yaml

```
@@ -9,12 +9,32 @@
   register: images_cache_find_result
   changed_when: false
 
-- name: import images from files in the local images cache  # noqa no-changed-when
-  shell:
-    executable: /bin/bash
-    cmd: ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }})
+- block:
+    - name: import images from files in the local images cache # noqa no-changed-when
+      shell:
+        executable: /bin/bash
+        cmd: ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }})
+      when:
+        - item.split('/')[-1] | regex_search('^1\.(\d)+\.(\d)+_images\.tar\.gz$')
+        - item.split('/')[-1].split('.')[1] |int < 24
+      register: ctr_images_import
+      failed_when: ctr_images_import.rc != 0
+      changed_when: ctr_images_import.rc == 0
+      with_items: "{{  images_cache_find_result.files | map(attribute='path') | list }}"
```

Users are already including additional bundles that they have built with `docker save` and may push mindthegap bundles either generated by DKP's release process or the dkp CLI, and these will not conform to any regex.

```suggestion
      when:
        mindthegap_images_import.rc != 0
      failed_when: False
      register: ctr_images_import
      changed_when: ctr_images_import.rc == 0
      with_items: "{{  images_cache_find_result.files | map(attribute='path') | list }}"
```

---

I think this block should also move after `mindthegap` since going forward we prefer `mindthegap` and don't want to run this task and fail every time.